Makefile

```diff
@@ -5,8 +5,8 @@ NAMESPACE=redhat-rhmi-operator
 PROJECT=integreatly-operator
 REG=quay.io
 SHELL=/bin/bash
-PREVIOUS_TAG ?=2.0.0
 TAG ?= 2.1.0
+SEMVER ?= 2.1.0-rc1
```

Comment:
Could TAG just be replaced entirely by SEMVER? There is no difference in the usage of it in this file, and TAG does not actually appear to represent a tag anyway since we apply the v prefix where it is actually used as a tag.

Reply:
Yeah it's used in the INTEGREATLY_OPERATOR_IMAGE var in the Makefile. That doesn't include the -rc* part of the semver, I don't know if it should though

Reply:
The problem here is that this variable was getting used for 2 different things, a VERSION during gen/scv and a TAG for some old commands.that built and pushed an image to quay.

Since nobody should be using those commands to push images to quay.io, i'd be in favour of removing them entirely, but that is probably out of scope of this work and i don't want to delay it any longer.

@philbrookes @maleck13 You any opinion on this? Does anyone actually use these commands https://github.com/integr8ly/integreatly-operator/blob/v2.1.0-er1/Makefile#L117-L130. They shouldn't be for anything going back to our quay registry anyway.

Reply:
I don't think they are used much anyway, I have no problem with these commands being modified.